### PR TITLE
Add command to reset a user's password and email it to them

### PR DESCRIPTION
**Pull request by penalosa**
Add a `resetpw` subcommand that resets a user's password and emails it to them

### CODE REVIEW

**Comment by AbstractBeliefs** [comment]
Can we look at if `passwd -e` works correctly on tardis? If so, we should do that to force a new password on the next login.

**Review by mayacopeland — COMMENTED** [review]
Doesn't this need a -?

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,5 @@ docs/_build/
 
 # PyBuilder
 target/
+
+ldap.conf
```
Comment (qaisjp) [inline .gitignore:65]: There should be a new line at the end of files

File: tardis/commands/resetpw.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import string
+import pprint
+import os
+from subprocess import Popen, PIPE
+
+
+# resetpw - Reset a user's password
+# exit codes:
+#   0   - Normal
+#   100 - User not found
```
Comment (AbstractBeliefs) [inline tardis/commands/resetpw.py:15]: Can we update this to use the error registry that's since landed?

**Review by AbstractBeliefs — CHANGES_REQUESTED** [review]
Once the error handling is updated I'm happy to approve this. I've seen it tested as well.

I still don't think emailing cleartext passwords is a good way forward but I can't think of anything else that is pragmatic just now. You need to have someone actively looking at #11 or we're going to end up with lots and lots of active passwords sitting in cleartext in inboxes just waiting to be abused, with no way for us to track or expire them.

**Comment by penalosa** [comment]
This is extremely out of date, so I'm going to close this. Feel free to pick it back up if it's useful!

**Pull request closed without merging**